From 2717375cb52e8d9d83067d1d696b1b04e418073d Mon Sep 17 00:00:00 2001
From: Vitaly Zaitsev <vitaly@easycoding.org>
Date: Wed, 21 Sep 2016 13:37:28 +0200
Subject: [PATCH] Build using systemwide cmake.

---
 Telegram/gyp/refresh.sh | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/Telegram/gyp/refresh.sh b/Telegram/gyp/refresh.sh
index 5a456ce..d436d58 100755
--- a/Telegram/gyp/refresh.sh
+++ b/Telegram/gyp/refresh.sh
@@ -10,9 +10,9 @@ cd $FullScriptPath
 if [ "$MySystem" == "Linux" ]; then
   ../../../Libraries/gyp/gyp --depth=. --generator-output=../.. -Goutput_dir=out Telegram.gyp --format=cmake
   cd ../../out/Debug
-  ../../../Libraries/cmake-3.6.2/bin/cmake .
+  cmake .
   cd ../Release
-  ../../../Libraries/cmake-3.6.2/bin/cmake .
+  cmake .
   cd ../../Telegram/gyp
 else
   #gyp --depth=. --generator-output=../.. -Goutput_dir=out Telegram.gyp --format=ninja
-- 
2.7.4

